### PR TITLE
Add quicq detection

### PR DESCRIPTION
**Pull request by gijsbotje**
adds quicq detection

quicq is an image optimization tool built by Afosto

examples:
https://www.chilicamper.nl/
https://www.coccinelle.nl/
https://www.azalp.nl/

### CODE REVIEW

File: src/technologies/q.json [hunk 1]
```diff
@@ -160,6 +160,23 @@
     "scriptSrc": "quicklink@([\\d.]+)/dist/quicklink.*\\.js\\;version:\\1",
     "website": "https://getquick.link/"
   },
+  "Quicq": {
+    "cats": [
+      31,
+      95,
+      9
+    ],
```
Comment (nurbek91) [inline src/technologies/q.json:168]: Let's use one category 
92 (Performance) 
Also  we can leave 31(CDN) probably

File: src/technologies/q.json [hunk 2]
```diff
@@ -160,6 +160,23 @@
...
+    "description": "The perfect image for all visitors and devices. Quicq's smart technology automatically switches between different next-gen image formats based on the visitor's device.",
+    "dom": "img[src*='.qcqcdn.com/'], img[data-src*='.qcqcdn.com/'], img[src*='cdn.quicq.io/'], img[data-src*='cdn.quicq.io/']",
+    "icon": "Quicq.svg",
+    "pricing": [
+      "freemium",
+      "mid",
+      "recurring",
+      "poa"
+    ],
```
Comment (nurbek91) [inline src/technologies/q.json:177]: I think it will suffice to indicate these
```
"freemium",
"payg"
```

File: src/technologies/q.json [hunk 3]
```diff
@@ -160,6 +160,23 @@
     "scriptSrc": "quicklink@([\\d.]+)/dist/quicklink.*\\.js\\;version:\\1",
     "website": "https://getquick.link/"
   },
+  "Quicq": {
+    "cats": [
+      31,
+      95,
+      9
+    ],
+    "description": "The perfect image for all visitors and devices. Quicq's smart technology automatically switches between different next-gen image formats based on the visitor's device.",
```
Comment (nurbek91) [inline src/technologies/q.json:169]: Something like this.
"description": "Quicq is an image optimisation app by Afosto.",

File: src/icons/Afosto.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 42 48"><g fill="none"><path fill="#04A54F" d="M41.48 35.9v11.83l-9.83-6.1"/><path fill="#2F3B60" d="M41.45 35.9l-20.7 11.96V23.94"/><path fill="#1F76BB" d="M41.45 12v23.9l-20.7-11.96"/><path fill="#8CC542" d="M41.45 11.97V12l-20.7 11.94V0"/><path fill="#E92629" d="M20.76 23.94v23.92h-.03L.03 35.93v-.02"/><path fill="#FAED20" d="M20.76 0v23.94L.03 11.97 20.73 0"/><path fill="#F15B2A" d="M20.76 23.94L.03 35.9v.02L0 35.9V11.97h.02"/></g></svg>
```
Comment (nurbek91) [inline src/icons/Afosto.svg:1]: Do we need an Afosto icon at this stage

File: src/apps.json [hunk 5]
```diff
@@ -0,0 +1,11213 @@
+{
```
Comment (nurbek91) [inline src/apps.json]: Do we need this file? It should probably be removed too.

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!
Small updates required to be merged

**Comment by nurbek91** [comment]
And the last request, could you share with examples in the PR description

File: src/icons/Afosto.svg (hunk 4 above)
Comment (gijsbotje) [reply src/icons/Afosto.svg:1]: Afosto itself is for detecting ecommerce sites built with our main software. Quicq is a tool we created for our users and others that don't use our ecommerce software.

**Comment by gijsbotje** [comment]
@nurbek91 updated the branch, sorry did it from an old master branch. Should be good now.
Also added examples.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**